examples/etherpad/Nulecule.yaml

```diff
@@ -0,0 +1,104 @@
+# Using etherpad as an example
```

Comment:
I have got this feeling that this file might get too large and unreadable pretty soon.